app/views/audit_mailer/summary.html.erb

```diff
@@ -0,0 +1,31 @@
+<body style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1; padding: 10px 24px">
+<h2 style="font-weight: normal; text-transform: uppercase; font-size: 120%;"><%= _("<b>Foreman</b> audit summary").html_safe %></h2>
+<ul style="list-style-type: none;">
+  <% if @audits.size > 0 %>
+    <%= n_('Displaying %{num_audits} of %{total_audits} audit',
+           'Displaying %{num_audits} of %{total_audits} audits', @count) % {:num_audits => @limit, :total_audits => @count} %>
```

Comment:
Okay, changed to:
```ruby
@count   = Audit.search_for(time).count
@limit   = Setting[:entries_per_page] > @count ? @count : Setting[:entries_per_page]
```
So now it will be "displaying 20 of 101 audits" (when audits on time are bigger than per_page)
Or "displaying 5 of 5 audits" (when audits on time are smaller)